s32/drivers/s32ze/HSE/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: BSD-3-Clause
+
+zephyr_include_directories(
```

Comment:
should be only included if crypto is built

Reply:
I updated